conf.d/android-sdk.fish

```diff
@@ -1,13 +1,18 @@
-set paths $ANDROID_HOME/{tool?,platform-tool?}
-  or set paths $ANDROID_SDK_ROOT/{tool?,platform-tool?}
-  or set paths $HOME/Android/Sdk/{tool?,platform-tool?}
-
-if test (count $paths) -gt 0
-  set -l IFS /
-  echo $paths[1] | read -la components
-  set -e components[-1]
-  printf '/%s' $components | read -l root
-
-  set -gx PATH $PATH $paths
-  set -gx ANDROID_HOME $root
+# check for common Android SDK locations
+for root in $ANDROID_SDK_ROOT $ANDROID_SDK_ROOT $HOME/Android/Sdk $HOME/Library/Android/sdk
```

Comment:
Is `$ANDROID_SDK_ROOT` repeated here on purpose?
```suggestion
for root in $ANDROID_SDK_ROOT $HOME/Android/Sdk $HOME/Library/Android/sdk
```

Reply:
Ah, I think the first one was supposed to be ANDROID_HOME